scripts/shared/lib/shflags

```diff
@@ -8,7 +8,7 @@ ensure_shflags() {
 	return
     fi
 
-    if [ ! -f .shflags ]; then
+    if [ ! -s .shflags ]; then
         echo "Downloading shflags ${SHFLAGS_VERSION}"
         if ! curl -L "https://raw.githubusercontent.com/kward/shflags/${SHFLAGS_VERSION}/src/shflags" > .shflags; then
             echo "$0 needs shflags in /usr/share/shflags/shflags or locally as .shflags." 1>&2
```

Comment:
For extra bonus points, we could delete `.shflags` if `curl` fails. The shell creates `.shflags` because of the redirection, and a failed `curl` leaves it empty (or containing nonsense).

```suggestion
            rm -f .shflags
            echo "$0 needs shflags in /usr/share/shflags/shflags or locally as .shflags." 1>&2
```

Reply:
Done, thanks!